backend/internal/application/service/applicationservice.go

```diff
@@ -48,6 +48,8 @@ func GetApplicationService() ApplicationServiceInterface {
 	return &ApplicationService{}
 }
 
+//var _ ApplicationServiceInterface = &ApplicationService{}
```

Comment:
Is this intentional?

Reply:
No, fixed with https://github.com/asgardeo/thunder/compare/ad3e9c6cce73984f7d28262c2d9b2be7ca033707..50bfe51c46429ed66b339843a0ba4eac32dca681